examples/distributed/intranode/example_alltoall.py

```
@@ -0,0 +1,102 @@
+import torch.distributed as dist
+import argparse
+
+
+def alltoall(PE_num, M, N, block_M, block_N, threads):
+    assert block_N == N
+
+    @T.prim_func
+    def main(
+            src: T.Tensor((PE_num * M, N), "float16"),
+            dst: T.Tensor((PE_num * M, N), "float16"),
+            barrier: T.Tensor((PE_num), "int32"),
```

Ruff (0.14.14)

[warning] Unused function argument: `barrier`

(ARG001)